soaktest/README.md

```diff
@@ -0,0 +1,28 @@
+# Information for soak test for go client
+The soak test contains 2 clients:
```

Comment:
Maybe "two applications" is more appropriate, since it is really 4 clients underneath